src/Icons/Icon.tsx

```diff
@@ -13,22 +13,26 @@ const IconWrapper = styled.div<{color:string}>`
 
 export {IconSVGs, IconWrapper};
 
+// Added Class name because close Icon seems not to be perfectly square
+// had to use flex to align it to its center but didn't wanted to do it for the base component
+
 interface IProps {
   icon: string;
   size?: number;
   weight?: 'light' | 'regular' | 'heavy'
   color?: ISvgIcons['color']
+  className?: string;
 }
 
-const Icon : React.FC<IProps> = ({icon, size = 24, weight = 'regular', color = 'mono'}) => {
+const Icon : React.FC<IProps> = ({icon, size = 24, weight = 'regular', color = 'mono', className}) => {
 
   const getKeyValue = <U extends keyof T, T extends object>(key: U) => (obj: T) => obj[key];
 
   const iconWeight : number = dimensions.icons.weights[weight];
   const IconSVG = getKeyValue<keyof any, any>(icon)(IconSVGs);
   if(icon in IconSVGs && IconSVG){
 
-    return <IconWrapper color={color}>
+    return <IconWrapper className={className} color={color}>
```

Comment:
I'm not sure this is the best way to solve this problem. It sounds like it could be a bug that needs fixing in SVG generation perhaps. Could you show me where you used this className prop in one of your fixes? I couldn't find it in the files.

As we're using StyledComponents I feel like it's best to try and avoid use of class names as it can open things up to problematic solutions and workarounds potentially.

Reply:
The problem was that the Div that is wrapping the icon is adding 3px height by default (no idea why 😞 )
Because of that I couldn't align it vertically. I added flex to the wrapper via className.

<img width="764" alt="Screen Shot 2021-02-25 at 18 08 56" src="https://user-images.githubusercontent.com/10409078/109132895-b24ba100-7797-11eb-93b0-6d10706ad910.png">

However I took a longer look to the Icon.tsx and notice that it's exporting the IconWrapper.
I understand that I should be using the exported wrapper to interact with it. 
I have removed the className option from the Icon.tsx

Reply:
Glad you could solve it this way. I'm sure there's some improvements we can make to make this process easier to understand and work with maybe